Serializer/JMSSerializerAdapter.php

```diff
@@ -96,6 +96,11 @@ private function convertContext(Context $context, $direction)
         if (null !== $context->getSerializeNull()) {
             $jmsContext->setSerializeNull($context->getSerializeNull());
         }
+        if (is_array($strategies = $context->getExclusionStrategies())) {
```

Comment:
On second thought `is_array ` and an exception is not needed, any invalid data will clearly make itself known when the foreach fails. Pushed a simplied version just now.